### PR TITLE
fonts settings UI changed

### LINKED ISSUE #2279: [Feature Request] Allow controls the font family
Insomnia Core & Designer Version: 2020.2.2
Operating System: Windows 10

# Description

Current preferences setting are only allow select one font style. It can not cover non-English characters.

image

Hope can add the feature to setting font family just like VS Code. 👍

image

### PR DESCRIPTION
**Pull request by sergiyladnych**
This PR Closes #2279. It was changed UI element to set Interface and Editor fonts from Dropdown to Text Input. Help texts were added for both.

|Before |After   |
|-------|-------|
| Screenshot before | Screenshot after |

### CODE REVIEW

**Comment by wdawson** [comment]
Thanks for the PR @sergiyladnych ! Could you please include some screenshots of the app before and after like in [this PR](https://github.com/Kong/insomnia/pull/4159)?

**Comment by sergiyladnych** [comment]
> Thanks for the PR @sergiyladnych ! Could you please include some screenshots of the app before and after like in [this PR](https://github.com/Kong/insomnia/pull/4159)?

Done. I have updated initial comment.

**Comment by sergiyladnych** [comment]
@develohpanda hello. I'm curious, is there any difficulties to do review of the PR?

**Review by develohpanda — APPROVED** [review]
Thanks for the PR! I tested and reviewed, and have made a couple of text updates to the tooltip messaging. In a future PR I think we can drop font-scanner entirely 🎉

File: packages/insomnia-app/app/ui/components/settings/general.tsx [hunk 1]
```diff
@@ -184,7 +184,6 @@ class General extends PureComponent<Props, State> {
 
   render() {
     const { settings } = this.props;
-    const { fonts, fontsMono } = this.state;
```
Comment (jackkav) [inline packages/insomnia-app/app/ui/components/settings/general.tsx:187]: Thanks @sergiyladnych As fonts, and fontsMono are removed in this PR and aren't used in the rest of this component or anywhere else in the code, the following lines should also be removed as part of this change.
- line: 2 import font-scanner
- line: 40-44 const
- line 52-62 state interface
- line 65-84 state declaration and componentDidMount
and finally
packages/insomnia-app/package.json
- line 95 font-scanner
the `npm run bootstrap` in the root folder to update the package lock

**Review by jackkav — CHANGES_REQUESTED** [review]

File: packages/insomnia-app/app/ui/components/settings/general.tsx (hunk 1 above)
Comment (sergiyladnych) [reply packages/insomnia-app/app/ui/components/settings/general.tsx:187]: @jackkav Thank you for feedback.
I have committed the changes

Comment (jackkav) [reply packages/insomnia-app/app/ui/components/settings/general.tsx:187]: awesome, can you also run `npm run bootstrap` in the root folder, to get the font-scanner out of the package lock

Comment (sergiyladnych) [reply packages/insomnia-app/app/ui/components/settings/general.tsx:187]: @jackkav I did run `npm run bootstrap` first time that you asked me. Now when I do that there is no difference/changes to commit. If you still think that I should do that, could you be more precise what is expected from that run.

Comment (jackkav) [reply packages/insomnia-app/app/ui/components/settings/general.tsx:187]: The package-lock.json should have updated and removed the references to font-scanner. I can take a look.

Comment (jackkav) [reply packages/insomnia-app/app/ui/components/settings/general.tsx:187]: my bad, it needed a `npm run clean && npm run bootstrap` I'll push the commit, we've been looking for a way to get rid of this binary dependency, should improve dx a lot! huge thanks!

**Review by jackkav — APPROVED** [review]
🚀

**Pull request merged**